source/develop/release-management/features/hosted-engine-indicate-host-in-web-ui.html.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Hosted Engine indicate hosted engine host in Web UI
+category: feature
+authors: pcbailey
+wiki_category: Feature|Hosted Engine indicate hosted engine host in Web UI
+wiki_title: Features/Hosted Engine indicate hosted engine host in Web UI
+wiki_revision_count: 0
+wiki_last_updated: 2016-04-19
+feature_name: 'Hosted Engine: indicate hosted engine host in Web UI'
+feature_modules: all
+feature_status: NEW
+---
+
+# Hosted Engine: Indicate Host Running the Hosted Engine VM
+
+### Summary
+
+Provide a visual indicator on the host running the hosted engine virtual machine in the hosts main tab of the administration portal user interface.
+
+### Owner
+
+*   Name: Phillip Bailey
+*   Email: phillip@redhat.com
+
+### Detailed Description
+
+*   The VDS business entity model must be updated to reflect whether it is hosting the hosted engine VM
+*   The ovirt-engine web UI must be changed to display the indicator on the hosted engine host
+
+### Benefit to oVirt
+
+*   A clear visual indicator will improve the user experience by adding to the amount of pertinent information available in the administration interface
+
+### Dependencies / Related Features
+
+\<tbd\>
+
+### Documentation / External References
+
+\<tbd\>
+
+### Testing
+
+\<tbd\>
```

Comment:
being a visual indication, I guess an initial testing may be:
- Deploy hosted engine on at least 2 hosts
- Check where the vm is running
- Check the web ui shows the indicator on the right host